hack/cri-reset.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+CRICTL_CLI="crictl --runtime-endpoint /var/run/cri-containerd.sock --image-endpoint --/var/run/cri-containerd.sock"
```

Comment:
Still only runtime-endpoint is needed. :)

Reply:
arghhh